### DISCUSSION TITLE
Is there a way to make a Whatsapp Channel forward messages to Discord?

### DISCUSSION THREAD

**Discussion by mondongoflow** [FKLC/WhatsAppToDiscord#216] [Q&A]
I tried to search for the channel name under list but it does not appear. I only see groups I'm joined in and my contacts yet I do not see Channels that I follow on WhatsApp as an option. Would be neat if I can set this up. If anyone has any idea on how to set this up then please let me know.

**Comment by malagoni** [comment]
WPP Channel messages are automatically sent to your WA2DC server, if you noticed it's disabled try using "enablePublishing" on control-room to turn it back on.
